pep/app/pods/components/search/form/component.ts

```diff
@@ -57,6 +60,10 @@ export default class SearchForm extends Component<SearchFormArgs> {
         );
     }
 
+    get hasEnteredLimits() {
```

Comment:
@billdami It doesn't look like `hasEnteredLimits` is being used anywhere currently. Should this be removed?